src/plugins/sqlalchemy_create_object_operator.py

```diff
@@ -0,0 +1,62 @@
+from airflow.models.baseoperator import BaseOperator
+
+from schematools.utils import schema_fetch_url_file
+from schematools.importer.base import BaseImporter
+from schematools.types import DatasetSchema, SchemaType
+from schematools.cli import _get_engine
+
+from environs import Env
+
+env = Env()
+# split is used to remove url params, if exists.
+# for database connection the url params must be omitted.
+default_db_conn = env("AIRFLOW_CONN_POSTGRES_DEFAULT").split("?")[0]
+
+
+class SqlAlchemyCreateObjectOperator(BaseOperator):
+    """ This operator takes a JSON data schema definition and DB connection to create the specified
+    tables and/or an index. The latter is based on the Identifier object in the data JSON schema i.e.
+    "identifier": ["identificatie", "volgnummer"] which is common for temporal data.
+    Note: The table creation alone leads to the creation of a PK which is already indexed by default.
+    Note2: A geometry column type, when specified in the JSON schema, also defaults to an index in the database.
+    """
+
+    def __init__(
+        self,
+        data_schema_name,
+        data_schema_env=None,
+        db_conn=None,
+        ind_table=None,
```

Comment:
Deze (en ook ind_identifier_index) zijn toch booleans? Is het dan niet logische om ze default `True` te geven? Dan heb je regel 39, 40 helemaal niet nodig.

Reply:
Je hebt gelijk. Dit zijn ook geen mutable objecten, dus dan is een directe default waarde prima. Ik pas het aan